manifest_staging/charts/secrets-store-csi-driver/README.md

```diff
@@ -52,11 +53,13 @@ The following table lists the configurable parameters of the csi-secrets-store-p
 | `linux.registrarImage.repository`       | Linux node-driver-registrar image repository                                                                                                                                   | `registry.k8s.io/sig-storage/csi-node-driver-registrar` |
 | `linux.registrarImage.pullPolicy`       | Linux node-driver-registrar image pull policy                                                                                                                                  | `IfNotPresent`                                          |
 | `linux.registrarImage.tag`              | Linux node-driver-registrar image tag                                                                                                                                          | `v2.8.0`                                                |
+| `linux.registrarImage.digest`              | Linux node-driver-registrar image digest                                                                                                                                          | `sha256:xxxxx`                                                |
```

Comment:
Same comments as above for all the values added:
- the default value is empty, so remove `sha256:...`
- Make the description clear that if digest is provided, it'll be used instead of tags